src/components/Expander/Expander.spec.jsx

```diff
@@ -0,0 +1,134 @@
+				assert.equal(mount(<Expander />).prop('onExpand'), _.noop);
+			});
+		});
+
+		describe('Label (as a prop)', () => {
+			it('renders the value in the header in a `SPAN` element neighboring its `ChevronIcon` instance.', () => {
+				assert.equal(
+					shallow(<Expander Label='foo' />)
+							.find(ReactCSSTransitionGroup)
+							.find('span')
+							.prop('children'),
+					'foo'
+				);
+			});
+		});
+
+		describe('Label (as a child)', () => {
+			it('renders the value in the header in a `SPAN` element neighboring its `ChevronIcon` instance.', () => {
+				const wrapper = shallow(
+					<Expander>
+						<Expander.Label>foo</Expander.Label>
+					</Expander>
+				);
+
+				assert.equal(
+					wrapper
+							.find(ReactCSSTransitionGroup)
+							.find('span')
+							.prop('children'),
+					'foo'
+				);
+			});
+		});
+
+		describe('pass throughs', () => {
+			it('passes through all props not defined in `propTypes` to the root element.', () => {
+				const wrapper = shallow(
+					<Expander
+							className='wut'
+							isExpanded={true}
+							onExpand={_.noop}
+							style={{ marginRight: 10 }}
+							foo={1}
+							bar={2}
+							baz={3}
+							qux={4}
+							quux={5}
+					/>
+				);
+				const rootProps = wrapper.find('.lucid-Expander').props();
+
+				// It should pass `foo`, `bar`, `baz`, `qux`, and `quux` through
+				// to the `Switch` instance.
+				_.forEach(['foo', 'bar', 'baz', 'qux', 'quux'], (prop) => {
+					assert(_.has(rootProps, prop));
+				});
+			});
+		});
+	});
+});
+
+describeWithDOM('Expander', () => {
+	describe('user clicks on the header', () => {
+		it('calls the function passed in as the `onExpand` prop...', () => {
+			_.forEach(['-header', '-icon', '-text'], (classSubstring) => {
+				const onExpand = sinon.spy();
+
+				mount(<Expander onExpand={onExpand} />).find(`.lucid-Expander${classSubstring}`).simulate('click');
+				assert(onExpand.calledOnce);
+			});
+		});
+
+		it('...and when `isExpanded` equals `false` passes along `true` as the first argument and a React synthetic event as the second argument.', () => {
+			_.forEach(['-header', '-icon', '-text'], (classSubstring) => {
+				const onExpand = sinon.spy();
+
+				mount(<Expander isExpanded={false} onExpand={onExpand} />).find(`.lucid-Expander${classSubstring}`).simulate('click');
+				assert.equal(onExpand.args[0][0], true);
+				assert(_.last(onExpand.args[0]).event instanceof SyntheticEvent);
+			});
+		});
+
+		it('...and when `isExpanded` equals `true` passes along `false` as the first argument and a React synthetic event as the second argument.', () => {
+			_.forEach(['-header', '-icon', '-text'], (classSubstring) => {
+				const onExpand = sinon.spy();
+
+				mount(<Expander isExpanded={true} onExpand={onExpand} />).find(`.lucid-Expander${classSubstring}`).simulate('click');
+				assert.equal(onExpand.args[0][0], false);
+				assert(_.last(onExpand.args[0]).event instanceof SyntheticEvent);
```

Comment:
I think the `instanceof` check isn't necesarry, just the fact that event was passed should be enough to make this test pass. Some components don't even pass SyntheticEvents here which is OK too. (Autocomplete's `onChange` is [one example](https://github.com/appnexus/lucid/blob/master/src/components/Autocomplete/Autocomplete.jsx#L189))